questions/The-List-Ordering-Mayhem.md

```diff
@@ -0,0 +1,36 @@
+# List Ordering Mayhem
+
+## Question
+
+Welcome to **List Ordering Mayham**! Here is the problem statement:
+
+> You are given an unordered list of the knights of players playing a chess game, each have their own x and y axes of the position they are on the board. Your task is to write a function that returns the correct descending order of y/number value in the list of the knights on the chessboard.
+
+
+## Inputs/Outputs
+Example Input:
+```python
+list = [[F,5], [G,2], [C,9], [A,7], [E,1]]
+```
+
+Expected Output:
+```python
+list = [[E,1], [G,2], [F,5], [A,7], [C,9]]
+```
+
+
+## Marking Criteria:
+- Achieve an efficient solution
+- Achieve an elegant / easy to read solution
+- Document your code ensuring good readability
+- Use good code design patterns and practicies within your code
+- Use good naming conventions throught your code
+- Attempt to get a relativly low time complexetiy if possible / O(log(n)) or O(n) is preferable
+
+
+## Example Code
+
+```python
+function generateOrderedList(list l):
+    >>>
+```
```

Comment:
Looks good! Could also be an interesting question when it comes to learning about objects/classes (e.g ordering the y-val then outputting the ordered x-vals or something like that)